src/firmware/app/world/firmware_world.c

```diff
@@ -32,3 +35,8 @@ FirmwareBall_t* app_firmware_world_getBall(FirmwareWorld_t* world)
 {
     return world->ball;
 }
+
+float app_firmware_world_getCurrentTime(FirmwareWorld_t* world)
+{
+    return world->get_current_time_seconds();
+}
```

Comment:
The language here seems to imply that it's some sort of timestamp related to when the world was received, but it's actually the rtos time. We could specify rtoscurrenttime, but I don't see what this function has to do with the world

Reply:
Hmm. So `FirmwareWorld` is intended to be the abstraction for the app layer to interact with the rest of the world. It should not matter _how_ we got the time (it could be RTOS time, could be something entirely different in sim). What are your thoughts on the way to best rephrase the language to make this clear?

Reply:
ok would a time since power on abstraction work in sim?